### PR TITLE
in-memory-object-store

### PR DESCRIPTION
**Pull request by Thomson42**
I'm not sure if I'm doing this right but here goes.

### CODE REVIEW

**Comment by nsmeds** [comment]
Your .travis.yml file is missing - make sure to include this in every project.

**Comment by nsmeds** [comment]
Travis still not passing - it looks like your .travis.yml is in your test folder. It needs to be in the root of your project.

**Comment by Thomson42** [comment]
It should be in the right space now, sorry.

**Comment by nsmeds** [comment]
Travis still failing. After submitting, keep an eye on the Travis check on this PR page to see if you get a green passing message or a red failing one. If it fails, take a look at the Travis build log to see what your errors are. Currently, I see the line `mocha not found` - I would check your package.json and make sure mocha is listed in your dependencies.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (nsmeds) [inline .vscode/launch.json]: You can add `.vscode` to your .gitignore file. These files shouldn't be committed.

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        assert.property(savedObj, '_id');
+        
+    });
+    it('returns `objectToSave` with added `_id` property', () => {
+        let store = new Store()
+        let savedObj = store.save({name : 'Tiffany'}); 
+        
+
+        assert.ok(savedObj._id);
+
```
Comment (nsmeds) [inline test/test.js]: These two tests are testing the same thing. Good to discover new assertion methods though.

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        let dog2 = store.save({name : 'Fido'});
+        let dog3 = store.save({name : 'Blaze'});
+        
+        let listCut = store.remove(dog2._id);
+        let listBite = store.getAll();
+        realList = store.list
+        console.log('listCut = ',listCut);
+        console.log('realList = ', realList);
+
+        assert.equal(listCut, listBite);
```
Comment (nsmeds) [inline test/test.js]: These two variables result from different function calls (`store.remove` vs `store.getAll()`), which return different things. It looks like your remove function works, but you'll need to modify your test assertion.

File: library/main.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    };
+    getAll(){
+        return this.list.slice();
+    };
+    //remove only gets rid of first element.
+    remove(id) {
+        //use indexof to find the right index for splice
+        var item = this.list.indexOf(id);
+        return this.list.splice(item,1);
+
```
Comment (nsmeds) [inline library/main.js]: This isn't what the [instructions](https://github.com/Thomson42/in-memory-object-store/blob/master/LAB.md) ask you to return.

**Pull request merged**